src/scripts/developer.ts

```diff
@@ -87,17 +78,15 @@ export function removeDeveloperByCondition({ devs, callbackCondition, args = []
 }
 
 export function sortDevelopersByEmploymentAndAge(developers: Developers, sortByAgeAscending: boolean): Developers {
-	const sortedDevs = developers.sort((firstDev: Developer, secondDev: Developer) => {
-		if (firstDev.isEmployed && secondDev.isEmployed) {
-			return sortByAgeAscending ? firstDev.age - secondDev.age : secondDev.age - firstDev.age;
-		}
+	const sortedDevs = developers.sort(
+		({ age: firstDevAge, isEmployed: firstDevIsEmp }: Developer, { age: secondDevAge, isEmployed: secondDevIsEmp }: Developer) => {
+			if (firstDevIsEmp && secondDevIsEmp) {
+				return sortByAgeAscending ? firstDevAge - secondDevAge : secondDevAge - firstDevAge;
+			}
 
-		return firstDev.isEmployed ? -1 : 1;
-	});
+			return firstDevIsEmp ? -1 : 1;
+		}
+	);
 
 	return sortedDevs;
 }
-
-export function addProperty<T extends object, K extends keyof any>(object: T, key: K, value: any): T {
```

Comment:
what happened to the implementation of this? why removed?

Reply:
I removed his function from developer.ts and added it in helpers/index.ts